ch-covidcertificate-backend-verifier/ch-covidcertificate-backend-verifier-sync/src/main/java/ch/admin/bag/covidcertificate/backend/verifier/sync/syncer/DgcRulesClient.java

```diff
@@ -11,25 +11,33 @@
 package ch.admin.bag.covidcertificate.backend.verifier.sync.syncer;
 
 import ch.admin.bag.covidcertificate.backend.verifier.model.sync.SigningPayload;
+import ch.admin.bag.covidcertificate.backend.verifier.sync.syncer.SigningClient.SigningException;
 import ch.admin.bag.covidcertificate.backend.verifier.sync.syncer.model.RulesSyncResult;
 import ch.admin.bag.covidcertificate.backend.verifier.sync.utils.CmsUtil;
+import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import java.nio.charset.StandardCharsets;
 import java.util.ArrayList;
 import java.util.Base64;
```

Comment:
I think this could be removed as well, since it is unused (according to sonarcloud)